src/controllers/OrderController.js

```diff
@@ -5,12 +5,21 @@ import updateOrder from '../utils/updateOrder';
 
 export class OrderController {
   makeOrder(request, response) {
-    OrderModel.makeOrder(request.body).then((result) => {;
+    OrderModel.makeOrder(request.body).then((result) => {
         response.status(200).json({ status: 'success', message: 'Order placed', data: result });
       });
     
   }
 
+  getOrderHistory(request, response) {
+    OrderModel.getOrderHistory(request.params.userId).then((result) => {
```

Comment:
You should validate this UserId before you make database calls.

Reply:
the userId <param> when not a valid UUID the database throws an error which I catch below at the catch block and return *invalid user ID*

Reply:
I think you should confirm the type of Id sent to avoid someone malicious injection SQL perhaps into your calls.

Reply:
ok